.openshift-ci/post_tests.py

```diff
@@ -68,17 +68,29 @@ class StoreArtifacts(RunWithBestEffortMixin):
     """For tests that only need to store artifacts"""
 
     def __init__(
-        self,
-        artifact_destination_prefix=None,
+            self,
+            artifact_destination_prefix=None,
     ):
         super().__init__()
         self.artifact_destination_prefix = artifact_destination_prefix
         self.data_to_store = []
 
-    def run(self, test_outputs=None):
+    def run(self, test_outputs=None, test_results=None):
         self.store_artifacts(test_outputs)
+        self.add_test_results(test_results)
         self.handle_run_failure()
 
+    def add_test_results(self, test_results):
+        if test_results is None:
```

Comment:
It's generally considered more pythonic to say:
```suggestion
        if not test_results:
```
This catches the empty list case too (and I guess it makes sense because we don't want to log the following line if we actually have nothing to store).